shuffle/cli/Cargo.toml

```diff
@@ -20,6 +20,8 @@ rand = "0.8.4"
 serde = { version = "1.0.124", features = ["derive"] }
 structopt = "0.3.21"
 toml = "0.5.8"
+ureq = { version = "1.5.4", features = ["charset", "json"], default-features = false }
```

Comment:
I recommend using reqwest over ureq

Reply:
Its a much better http client and can be used both in async and in sync contexts